en/controllers/request-response.rst

```diff
@@ -1094,6 +1094,11 @@ criteria are met:
 #. The request has an ``Origin`` header.
 #. The request's ``Origin`` value matches one of the allowed Origin values.
 
+.. tip::
+
+    CakePHP has no built-in CORS middleware! We recommend you build your
```

Comment:
This should also mention "why" we don't have one.

Reply:
I hope you are happy with the reason now 😄

Reply:
I hope this will reduce support questions :)